main/system.h

```diff
@@ -71,7 +71,6 @@
 #endif
 
 #ifdef IS_ATHENA
-#define HAS_SD_CARD 1
```

Comment:
I would have said that Athena does in fact have an SD card, or at least the holder for the card. If you have some weird Athena that doesn't have an SD card then you should change the definition locally.

Reply:
I would counter argue in that most people who use athena don't use the SD card, like the embedded systems team. This compiler flag was intended to be a CCS predefined symbol